### ISSUE TITLE
version in theme header

### ISSUE THREAD

**Issue by jbleys** [uweb/uw-2014#185]
Hi,
Looks like the `Version` in the style theme header didn't get updated to 3.0.0

Thanks,
Jasper
Department of Global Health

**Comment by lcaple** [comment]
Jasper, can you offer specifics about where this issue is and how to reproduce it? Thanks.

**Comment by jbleys** [comment]
Yes, I check the theme status on my site and update accordingly...

![image](https://user-images.githubusercontent.com/2615272/148307401-1a7ba706-7a50-47fb-8723-10590cd4108c.png)

But upon checking the stylesheet it's still the old version...

![image](https://user-images.githubusercontent.com/2615272/148307494-a2c4249e-4a5f-4a62-8340-794352f4bc61.png)

Leading me to think that the version in style.css wasn't updated.

But! I just checked the file here on github and that does look like it was updated.
So now I'm even more confused, ha. :)
And thus on second thought, whatever the issue is, it may not be an issue in the uw-2014 theme?

**Comment by dghweb** [comment]
Hi @lcaple,

This is why I missed the updated version. The release number 3.0.2 is not updated in the theme header. When I check the theme status on a website it does not show there's an update available.

Thanks,
Jasper Bleijs
Department of Global Health
